src/ORM/Table.php

```diff
@@ -1512,6 +1513,24 @@ public function save(EntityInterface $entity, $options = [])
     }
 
     /**
+     * Try to save an entity and fail if save was not successful
+     *
+     * @param \Cake\Datasource\EntityInterface $entity the entity to be saved
+     * @param array|\ArrayAccess $options The options to use when saving.
+     * @return \Cake\Datasource\EntityInterface
+     * @throws \RuntimeException When the entity couldn't be saved
```

Comment:
please add a `@see Table::save()` so people can jump to the documentation fir saving